packages/starlight/package.json

```diff
@@ -157,6 +157,7 @@
     "./props": "./props.ts",
     "./schema": "./schema.ts",
     "./types": "./types.ts",
+    "./expressive-code": "./integrations/expressive-code.ts",
```

Comment:
Do we really need to export this? What would be the use case?

If this is on purpose and I am missing something, I guess it should be documented as it exposes the entirety of `astro-expressive-code` and a few other functions?

Reply:
Well spotted! The use case is to provide users with all the classes and functions required for customization, without having to install `astro-expressive-code` as an additional dependency (and thereby risiking version conflicts). What users can do with the exports can be as simple as instantiating their own themes (`ExpressiveCodeTheme`), but can also become more involved to e.g. create own renderers (`createRenderer`) to be used for the `customCreateRenderer` option, or access and modify the texts of plugin-frames (`pluginFramesTexts`).

Reply:
Regarding your idea of documenting the export, do you have any specific location in mind where you would most likely search for something like this? I just had a look around the codebase and couldn't find any documentation on the existing exports, so we might need to add an entirely new section somewhere if we really consider this to be worth documenting. What do you think?

Reply:
I personally would say maybe adding an Expressive Code section in the [customization guide](https://starlight.astro.build/guides/customization/), kinda like edit links which have an [entry](https://starlight.astro.build/guides/customization/#edit-links) in this guide and in the configuration reference [too](https://starlight.astro.build/reference/configuration/#editlink). I also guess we would not need to detail everything as we can link to the Expressive Code documentation from there and only give an overview of what it is possible to do with it.

But maybe it's worth gathering more opinions on this before pulling the trigger.

Reply:
Resolved! Now exporting a dedicated `exports` file instead.